backend/src/main/java/pt/ua/deti/ies/SmartHomes/backend/WebSocket/WebSocketSender.java

```diff
@@ -0,0 +1,18 @@
+package pt.ua.deti.ies.SmartHomes.backend.WebSocket;
```

Comment:
You can keep this class if you want, but I don't think this is needed, because you're doing the same in the `Listener` class